x-pack/plugins/triggers_actions_ui/public/application/app.tsx

```diff
@@ -62,9 +66,24 @@ export const AppWithoutRouter = ({ sectionsRegex }: { sectionsRegex: string }) =
   const DEFAULT_SECTION: Section = canShowAlerts ? 'alerts' : 'connectors';
   return (
     <Switch>
-      <Route path={`${BASE_PATH}/:section(${sectionsRegex})`} component={TriggersActionsUIHome} />
-      {canShowAlerts && <Route path={routeToAlertDetails} component={AlertDetailsRoute} />}
+      <Route
+        path={`${BASE_PATH}/:section(${sectionsRegex})`}
+        component={suspendedRouteComponent(TriggersActionsUIHome)}
+      />
+      {canShowAlerts && (
+        <Route path={routeToAlertDetails} component={suspendedRouteComponent(AlertDetailsRoute)} />
+      )}
       <Redirect from={`${BASE_PATH}`} to={`${BASE_PATH}/${DEFAULT_SECTION}`} />
     </Switch>
   );
 };
+
+function suspendedRouteComponent<T = unknown>(
+  RouteComponent: React.ComponentType<RouteComponentProps<T>>
+) {
+  return (props: RouteComponentProps<T>) => (
+    <Suspense fallback={<EuiLoadingSpinner />}>
```

Comment:
Can we move the spinner to the middle and maybe bigger?

<img width="1360" alt="Screen Shot 2020-05-07 at 9 02 59 AM" src="https://user-images.githubusercontent.com/3694571/81298100-4cdcc700-9042-11ea-81b1-8d134fc99a7f.png">